tests/playwright/ui-tests/searchpage.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { test } from '@playwright/test'
+import { EndpointFeature, IWireMockFeatures, IWireMockRequest, IWireMockResponse, WireMock } from 'wiremock-captain'
+import { HomePage } from '../page-object-model/home-page'
+import { SearchPage } from '../page-object-model/search-page'
+import { DetailsPage } from '../page-object-model/trust/details-page'
+
+test.describe('homepage', () => {
+  let homePage: HomePage
+  let searchPage: SearchPage
+
+  const mock = new WireMock(process.env.WIREMOCK_BASEURL ?? 'http://localhost:8080')
+
+  const trustsResponseData = [
+    { GroupName: 'trust 1', TrustAddress: { Street: '12 Paddle Road', Locality: 'Bushy Park', AdditionalLine: 'Letworth', Town: 'Manchester', Postcode: 'MX12 P34' }, Ukprn: '123', Establishments: [{ urn: '123' }] },
+    { GroupName: 'trust 2', TrustAddress: { Street: '12 Paddle Road', Locality: '', AdditionalLine: '', Town: 'Manchester', Postcode: 'MX12 P34' }, Ukprn: '124', Establishments: [{ rn: '456' }, { urn: '789' }] },
+    { GroupName: 'trust 3', TrustAddress: { Street: '', Locality: 'Bushy Park', AdditionalLine: null, Town: 'Manchester', Postcode: '' }, Ukprn: '125', Establishments: [] }
+  ]
+
+  test.beforeAll(async () => {
+    const request: IWireMockRequest = {
+      method: 'GET',
+      endpoint: '/v3/trusts'
+    }
+
+    const mockedResponse: IWireMockResponse = {
+      status: 200,
+      body: {
+        Data: trustsResponseData
+      }
+    }
+
+    const features: IWireMockFeatures = { requestEndpointFeature: EndpointFeature.UrlPath }
+
+    await mock.register(request, mockedResponse, features)
+  })
+
+  test.beforeEach(async ({ page }) => {
+    homePage = new HomePage(page)
+    searchPage = new SearchPage(page)
+    await homePage.goTo()
+  })
+
+  test('Searching for a trust from the homepage displays formatted information about each trust', async () => {
+    await searchPage.goToSearchFor('trust')
+    await searchPage.expect.toBeOnPageWithResultsFor('trust')
+
+    await searchPage.expect.toShowResultsWithCount(3)
+
+    await searchPage.expect.toShowResultWithText('trust 1', '12 Paddle Road, Bushy Park, Letworth, Manchester, MX12 P34')
```

Comment:
it would be good to bundle the assertions for each trust together into a single assertion/assertion chain which also checks the unique reference number(s) displayed